goson.go

```
@@ -112,6 +115,25 @@ func GenerateJSONSchema(jsonStr string) (MapString, error) {
 	return res, nil
 }
 
+func GenerateJSONSchemaFromURL(url string) (MapString, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	defer resp.Body.Close()
+	body, _ := ioutil.ReadAll(resp.Body)
```

please don't ignore the error